pymc4/gp/cov.py

```diff
@@ -149,9 +189,11 @@ class CovarianceProd(Combination):
     Parameters
     ----------
     feature_ndims : int
-        number of rightmost dims to include in kernel computation
+        The number of rightmost dimensions to be absorbed during
```

Comment:
nitpicking. The line-length limit is 100.

Reply:
true! I am somehow always a little scared to exceed the limit. will fix that all over the module